packages/api/test/e2e/rpc.decodeResponse.e2e.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019-2020 Centrality Investments Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { RpcCoder } from "@polkadot/rpc-provider/coder";
```

Comment:
Used this from polkadot/rpc-provider at 2.7.1 version... when they upgrade and change.. we can still have it as a part of our test suite so that we are aware of what has changed..

Reply:
This is a test of the decoder itself. What we want is more like an e2e test. Like the result of a query before any decoding as a string of bytes.

Reply:
this will tell us if the behaviour of @rpc-provider changes but doesn't tell us if the behaviour of the node changes between versions. Can we run some of these queries against a node? e.g simple http request and verify response